server/memstore_test.go

```diff
@@ -1222,40 +1223,37 @@ func TestMemStoreMessageTTL(t *testing.T) {
 	require_Equal(t, ss.Msgs, 0)
 }
 
-func TestMemStoreSubjectDeleteMarkers(t *testing.T) {
```

Comment:
This non-skipped test should have stayed :sweat_smile: (also exists for the filestore)

Reply:
ok will add back in and force push last CL.